aries_cloudagent/ledger/routes.py

```diff
@@ -181,6 +224,7 @@ async def register_ledger_nym(request: web.BaseRequest):
         request: aiohttp request object
     """
     context: AdminRequestContext = request["context"]
+    outbound_handler = request["outbound_message_router"]
     async with context.profile.session() as session:
         ledger = session.inject_or(BaseLedger)
```

Comment:
Hm, it's not updated by this PR, but BaseLedger should be injected via the profile not the session. Generally anything injected by the session shouldn't be touched after the async with block as it might refer to a closed session. This is also true for the IndyLedgerRequestExecutor instances which seem to be set up by the conductor.

Reply:
OK thanks for the tip!